CONTRIBUTING.md

```diff
@@ -86,14 +86,22 @@ $ yarn
 
 ## Testing
 
-Before running _plugin_ tests, the data stores need to be running.
-The easiest way to start all of them is to use the provided
-docker-compose configuration:
-
-```sh
-$ docker-compose up -d -V --remove-orphans --force-recreate
-$ yarn services
-```
+### Prerequisites
+
+Before running _plugin_ tests, the data stores need to be running. And for that to work you first need to authenticate Docker against the GitHub Container Registry.
+
+1. [Create a new classic GitHub Personal Access Token (PAT)](https://github.com/settings/tokens/new) with `read:packages` permissions.
+2. [Authorise the new PAT with the Datadog org](https://docs.github.com/en/enterprise-cloud@latest/authentication/authenticating-with-saml-single-sign-on/authorizing-an-ssh-key-for-use-with-saml-single-sign-on).
```

Comment:
Does this mean only Datadog employees can run the plugin tests?

Reply:
If so we should probably explicitly tell the reader... I suppose we must have made a mistake at some point and required an internal container for testing?

Reply:
Good question. I actually haven't tried to do these steps without authorizing with the Datadog org. Maybe it's not required. I'll check

Reply:
I'm after testing this I found that authorizing with the PAT with the Datadog org _is_ required. But it's probably only the `couchbase-server-sandbox` package that requires it. It might be interesting to see if we could remove that requirement. But for now this is a required step. I'll update the text to explicitly reflect this.

Reply:
I've added a new commit with a mention of this

Reply:
Actually, after a bit of more investigation, the entire need for the PAT is becuase of `couchbase-server-sandbox`. If I remove that from the `docker-compose.yml` file, docker works just fine without being logged in.

Reply:
@bengl and I just discovered the culprit. While the repo for `couchbase-server-sandbox` is public, the package itself published under the repo isn't. We're just investigating if we can make it public. If so, this whole PR becomes irrelevant 🎉

Reply:
The package should be public now.